mozzi_midi.h

```diff
@@ -113,21 +113,34 @@ inline Q16n16  Q16n16_mtof(Q16n16 midival_fractional)
 	return (Q16n16) (freq1+ diff_fraction);
 };
 
-inline UFix<16,16> mtof(UFix<16,16> midival)
+/** @ingroup midi
+Converts midi note number with speed and accuracy from a UFix<16,16>.
+Uses Q16n16_mtof internally.
+*/
+template <uint64_t RANGE>
+inline UFix<16,16> mtof(UFix<16,16,RANGE> midival)
 {
   return UFix<16,16>::fromRaw(Q16n16_mtof(midival.asRaw()));
-}
+};
 
-template<int8_t NI, int8_t NF>
-  inline UFix<16,16> mtof(UFix<NI,NF> midival)
+/** @ingroup midi
+Converts midi note number with speed and accuracy from any UFix.
+Uses Q16n16_mtof internally.
```

Comment:
Optimization: A specialization for the case of plain midi notes (midival with NF=0) may avoid unexpeced inefficiency in many cases.

Reply:
I'll look it up. I might go through an optimization with Q8n0, that would be call upon (in the same spirit than what is done here with Q16n16).

Reply:
Done